docs/model-dev-guide/apis-howto/api-pytorch-ug.rst

```diff
@@ -50,6 +50,9 @@ a skeleton to keep track of what is still required. A good starting template can
        def evaluate_batch(self, batch: TorchData) -> Dict[str, Any]:
            return {}
 
+When running your experiment, you can specify the predefined :ref:`PyTorch Distributed Launcher
+<pytorch-dist-launcher>`.
+
```

Comment:
This needs to be way "louder".  Like, a new subheader before `Download Data` that is called `Distributed Backend`, which explains that Horovod is the default backend, but you can choose to use `torch.distirbuted` and `DistributedDataParallel` as your distributed backend, following <link to instructions>.